src/OmniSharp.Cake/CakeProjectSystem.cs

```diff
@@ -292,7 +306,8 @@ private ProjectInfo GetProject(CakeScript cakeScript, string filePath)
                 metadataReferences: GetMetadataReferences(cakeScript.References),
                 // TODO: projectReferences?
                 isSubmission: true,
-                hostObjectType: hostObjectType);
+                hostObjectType: hostObjectType)
+                .WithAnalyzerConfigDocuments(analyzerConfigDocuments);
```

Comment:
I think the same should be applied here https://github.com/OmniSharp/omnisharp-roslyn/blob/3163c234bc0cb2ece67624791a66d735b295b907/src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfoExtensions.cs#L61 as well?